src/openrct2/CmdlineSprite.cpp

```diff
@@ -130,13 +130,14 @@ static bool sprite_file_open(const utf8* path)
         }
 
         int32_t entryTableSize = spriteFileHeader.num_entries * sizeof(rct_g1_element);
-        spriteFileEntries = (rct_g1_element*)malloc(entryTableSize);
+        spriteFileEntries = static_cast<rct_g1_element*>(malloc(entryTableSize));
```

Comment:
There's not any point anymore to use malloc over new. This would get rid of the need for the cast. Guess it's slightly out of the scope of this pr but might be an idea to open an issue for it so that it gets tracked